manifests/07-downloads-helm.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: helm-download-links
+spec:
+  description: |
+    Helm 3 is a package manager for Kubernetes applications which enables defining,
+    installing and upgrading applications packaged as Helm Charts.
+    Helm 3 is in <a href=https://access.redhat.com/support/offerings/techpreview>Technology Preview</a> feature on OpenShift 4.
```

Comment:
Predrag, have you verified if it shows up as a clickable URL?

Reply:
Yes, see the screenshot attached above,

Reply:
either:
`Helm 3 is in Technology Preview on OpenShift`
or
`Helm 3 is a Technology Preview feature on Openshift`

but `Helm 3 is in Technology Preview feature on OpenShift` is not a grammatically correct sentence.

Reply:
There is also an e2e suite in console that verifies the CLIDownload links.  You could add a few lines here as well:

https://github.com/openshift/console/blob/e3080430f4ac297619b57062410c461333e62c9f/frontend/integration-tests/tests/crd-extensions.scenario.ts#L15